File: Assets/Bundles/Server/Resources/TS/Services/Global/Chat/Commands/TeamCommand.lua
-- Compiled with unity-ts v2.1.0-75
local ChatCommand = require("Server/TS/Services/Global/Chat/Commands/ChatCommand").ChatCommand
local TeamCommand
do
	local super = ChatCommand
	TeamCommand = setmetatable({}, {
		__tostring = function()
			return "TeamCommand"
		end,
		__index = super,
	})
	TeamCommand.__index = TeamCommand
	function TeamCommand.new(...)
		local self = setmetatable({}, TeamCommand)
		return self:constructor(...) or self
	end
	function TeamCommand:constructor()
		super.constructor(self, "team")
	end
	function TeamCommand:Execute(player, args)
		local team = player:GetTeam()
		if not team then
			player:SendMessage("You are not on a team.")
			return nil
		end
		player:SendMessage("You are on Team [" .. (team.id .. "]"))
	end
end
return {
	TeamCommand = TeamCommand,
}
-- ----------------------------------
-- ----------------------------------

File: Assets/Typescript~/src/Server/Services/Global/Chat/Commands/TeamCommand.ts
import { ChatCommand } from "Imports/Core/Shared/Commands/ChatCommand";
import { Player } from "Shared/Player/Player";

export class TeamCommand extends ChatCommand {
	constructor() {
		super("team");
	}
	public Execute(player: Player, args: string[]): void {
		const team = player.GetTeam();
		if (!team) {
			player.SendMessage("You are not on a team.");
			return;
		}

		player.SendMessage(`You are on Team [${team.id}]`);
	}
}
